apps/console/src/features/users/utils/generate-password.utils.ts

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+const generateRandomNumbers = (upperLimit: number): number => {
```

Comment:
Lets add doc comments to the functions.

Reply:
Fixed with f1efba513d6385145463c2f9f6462bf4a21f6b17